action.yml

```diff
@@ -51,6 +51,21 @@ inputs:
       Name of the base branch, where changes in package.json must be applied. It is used in PR creation. If not provided, default branch is used
       In the format: `next-major`.
     required: false
+  packagejson_path_list:
```

Comment:
what is your idea on how is this going to be used now?

atm we have generic worflow https://github.com/asyncapi/.github/blob/master/.github/workflows/bump.yml that is pushed to repos with https://github.com/asyncapi/.github/blob/master/.github/workflows/global-replicator.yml#L120 topic (25 different repos: https://github.com/topics/get-global-node-release-workflows)

below new introduced options somehow suggests me that you plan on introduction alternative workflow? or we will make `search: true` for all?

Reply:
Yep this can be done, should I merge these two then?